blocks/explore-articles/explore-articles.js

```diff
@@ -34,8 +35,10 @@ const getOptions = (list, placeholder) => {
   list.unshift(placeholder);
   list.forEach((el) => {
     const option = createElement('option', { props: { value: el } });
-    option.innerText = el;
-    if (el.length !== 0) options.push(option);
+    option.innerText = capitalizeWords(el);
```

Comment:
What is the purpose of the `capitalizeWords` function? Is it intended to ensure strings can be compared with consistent casing?

Reply:
no, in this case is only for the displaying string so that the select can be 'Mack Anthem' instead of 'mack anthem'. Unless there is a way we can change this with css its just a cosmetic thing